packages/sdk/src/_tests/webhook-handler.test.ts

```diff
@@ -0,0 +1,128 @@
+import { LinearWebhookClient, LINEAR_WEBHOOK_SIGNATURE_HEADER } from "../webhooks";
+import http from "http";
+import crypto from "crypto";
+import getPort from "get-port";
+import express from "express";
+import { Response } from "node-fetch";
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+(globalThis as any).Response = Response;
+
+export interface SignedBody {
+  body: Buffer;
+  signature: string;
+}
+
+describe("webhooks handlers", () => {
+  it("works with a fetch-style handler", async () => {
+    const secret = "SECRET";
+    const client = new LinearWebhookClient(secret);
+    const handler = client.createHandler();
+
+    const p = { ...samplePayload, webhookTimestamp: Date.now() };
+    const { body, signature } = createSignedBody(secret, p);
+
+    const headers = new Map<string, string>([
+      [LINEAR_WEBHOOK_SIGNATURE_HEADER, signature],
+      ["content-type", "application/json"],
+    ]);
+    const reqLike = {
+      method: "POST",
+      headers: { get: (key: string) => headers.get(key.toLowerCase()) ?? null },
+      arrayBuffer: async () => body,
+    } as unknown as Request;
+
+    let handled = false;
+    handler.on("*", evt => {
+      expect(evt.action).toBe("create");
+      handled = true;
+    });
+    const res = await handler(reqLike);
+    expect(res.status).toBe(200);
+    expect(await res.text()).toBe("OK");
+    expect(handled).toBe(true);
+  });
+
+  it("works with a express server", async () => {
+    const secret = "SECRET";
+    const client = new LinearWebhookClient(secret);
+    const handler = client.createHandler();
+    const app = express();
+    const port = await getPort();
+    let handled = false;
+    handler.on("*", evt => {
+      expect(evt.action).toBe("create");
+      handled = true;
+    });
+    app.post("/", (req, res) => {
+      void handler(req, res);
+    });
+    const server = app.listen(port);
+    try {
+      const p = { ...samplePayload, webhookTimestamp: Date.now() };
+      const { body, signature } = createSignedBody(secret, p);
+      const result = await httpPost(port, "/", { [LINEAR_WEBHOOK_SIGNATURE_HEADER]: signature }, body);
+      expect(result.status).toBe(200);
+      expect(result.text).toBe("OK");
+      expect(handled).toBe(true);
+    } finally {
+      server.close();
+      handler.removeAllListeners();
+    }
+  });
+});
+
+export function createSignedBody(secret: string, payload: Record<string, unknown>): SignedBody {
+  const body = Buffer.from(JSON.stringify(payload));
+  const signature = crypto.createHmac("sha256", secret).update(body).digest("hex");
```

Comment:
nit: I'm seeing several TS errors locally. Is that actually indicative of a problem we should care about? 

<img width="2062" height="956" alt="Image" src="https://github.com/user-attachments/assets/a1ff781b-66f1-4866-ba63-59fc7bfed36a" />

Reply:
Resolved- 4.7.4 is our pinned TS version for this repo